src/Api/SecretsManager/Models/Response/OrganizationCountsResponseModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Bit.Core.Models.Api;
+
+namespace Bit.Api.SecretsManager.Models.Response;
+
+public class OrganizationCountsResponseModel() : ResponseModel(_objectName)
+{
+    private const string _objectName = "organization_counts";
+
+
+    public int Projects { get; set; }
+
+    public int Secrets { get; set; }
+
+    public int MachineAccounts { get; set; }
```

Comment:
```suggestion
    public int ServiceAccounts { get; set; }
```

So far in the code base we are consistent with `ServiceAccounts` and just alter for user facing results.

Reply:
Fixed in https://github.com/bitwarden/server/commit/41b166c6059a33ccd52066d0b524f387254ffa0d